Package.swift

```diff
@@ -407,6 +410,13 @@ let package = Package(
             ],
             plugins: [swiftlintPlugin]
         ),
+        .testTarget(
+            name: "CrashesTests",
```

Comment:
There's a single test case that I moved from iOS because it didn't depend on the iOS app at all.